File: SWIFT/Cap05/c05ex10.swift
// c05ex10.swift

import Foundation

var RESP: Character = "S"

repeat {
    print("Entre um valor numerico: ", terminator: "")
    if let inputN = readLine(), let N = Int(inputN) {
        let R = N * 3
        print("Resultado = \(R)\n")

        print("Deseja continuar? (S/N): ", terminator: "")
        if let inputRESP = readLine(), 
           let charRESP = inputRESP.uppercased().first {
            RESP = charRESP
        }
    } else {
        print("\nEntrada invalida.\n")
        continue       
    }
    print()
    if (RESP.uppercased() != "S") {break}
} while (true)

print("\nTecle <Enter> para encerrar... ", terminator: "")
_ = readLine()

File: PYTHON/Cap05/c05ex10.py
# c05ex10.py

resp = "S"
while True:
  n = int(input("Entre um valor numerico: "))

  r = n * 3
  print("Resultado =", r)
  print()

  resp = input("Deseja continuar? (S/N): ").upper()[0]

  if (resp.upper() != "S"):
    break

  print()

print()  
input("\nTecle <Enter> para encerrar... ")
